apps/admin-gui/src/app/app.component.html

```diff
@@ -3,7 +3,7 @@
   {{ warningMessage }}
 </div>
 <perun-web-apps-notificator [displayWarning]="displayWarning"></perun-web-apps-notificator>
-<ng-template [ngIf]="!isLoginScreenShow && !isServiceAccess && !isServiceLogin()">
+<ng-template [ngIf]="!isLoginScreenShow && !isServiceAccess && !isServiceLogin() && !isLoggedIn()">
```

Comment:
Please don't use function call in the html template, if some calculation needs to be made use a pipe.

Reply:
I have changed this to a pipe call. However, this change would not allow removing the sidebar and information about logged-in user (when revoking the tokens).